src/main/resources/liquibase.properties

```diff
@@ -0,0 +1,5 @@
+url=jdbc:h2:mem:testdb
+username=root
+password=1234
+changeLogFile=db/changelog/db.changelog-master.yaml
+driver=org.h2.Driver
```

Comment:
from your app properties:
```
spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
```

but here you have `driver=org.h2.Driver`

is it okay ?)

Reply:
No, but my project can't be builded in github if I leave the settings for mysql. We talked about it with Lilia on QnA and she said to leave it like that for now